recipes/r-som/meta.yaml

```diff
@@ -0,0 +1,78 @@
+{% set version = '0.3-5.1' %}
+
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-som
+  version: {{ version|replace("-", "_") }}
+
+source:
+  fn: som_{{ version }}.tar.gz
+  url:
+    - https://cran.r-project.org/src/contrib/som_{{ version }}.tar.gz
+    - https://cran.r-project.org/src/contrib/Archive/som/som_{{ version }}.tar.gz
+
+
+  sha256: a6f4c0e5b36656b7a8ea144b057e3d7642a8b71972da387a7133f3dd65507fb9
+  # patches:
+   # List any patch files here
+   # - fix.patch
+
+build:
+  # If this is a new build for the same version, increment the build number.
+  number: 0
+
+  # This is required to make R link correctly on Linux.
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+
+requirements:
+  build:
+    - r-base
+    - posix                # [win]
+    - {{native}}toolchain  # [win]
+    - gcc                  # [not win]
+
+  run:
+    - r-base
```

Comment:
can you include `libgcc` here and clean up the files a little bit to remove the comments